tests/base/__init__.py

```diff
@@ -37,20 +37,24 @@ def override_global_config(values):
             >>> with self.override_global_config(dict(name=value,...)):
                 # Your test
         """
-        # DEV: Uses dict as interface but internally handled as attributes on Config instance
-        analytics_enabled_original = ddtrace.config.analytics_enabled
-        report_hostname_original = ddtrace.config.report_hostname
-        health_metrics_enabled_original = ddtrace.config.health_metrics_enabled
-
-        ddtrace.config.analytics_enabled = values.get('analytics_enabled', analytics_enabled_original)
-        ddtrace.config.report_hostname = values.get('report_hostname', report_hostname_original)
-        ddtrace.config.health_metrics_enabled = values.get('health_metrics_enabled', health_metrics_enabled_original)
+        global_config_keys = [
```

Comment:
some really brief comments here would help with readability since it is no longer immediately obvious what we're doing here